src/components/controls/km/km-validation-errors.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <div class="scbd-controls km-validation-errors document-validation">
+    <div
+      v-if="reports && reports.isAnalyzing"
+      class="alert alert-info"
+      role="alert"
+    >
+      <i class="bi bi-gear"></i> {{ t("analyzingDocument") }}
+    </div>
+    <div v-if="reports && reports.isSaving" class="alert alert-info" role="alert">
+      <i class="bi bi-gear"></i> {{ t("savingDocument") }}
+    </div>
+    <div
+      v-if="reports && !reports.isAnalyzing && !reports.isSaving && !reports.errors"
+      class="alert alert-success"
+      role="alert"
+    >
+      {{ t("valid") }}
+    </div>
+    <div v-if="reports && !reports.isAnalyzing && !reports.isSaving">
+      <div
+        v-if="
+          !hideErrors && reports && reports.errors && reports.errors.length
+        "
+        class="alert alert-warning"
+        role="alert"
+      >
+      
+        <button
+          type="button"
+          @click="onButtonClose"
+          aria-label="Close"
+          class="
+            position-absolute
+            top-0
+            end-0
+            m-2
+            bg-transparent
+            border-0
+            fs-5
+            text-secondary
+          "
+        >
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <strong>
+          {{ t("contains") }} {{ reports.errors.length }} {{ t("errors") }}
+        </strong>
+
+        <ul>
+          <li
+            v-for="error in reports.errors"
+            :title="error.parameters"
+            :key="error"
+          >
+            <span>{{ t(getTranslation(error)) }}</span>
+            <ul v-if="error.properties">
+              <li
+                v-for="property in error.properties"
+                :title="error.parameters"
+                :key="property"
+              >
+                <a
+                  class="text-decoration-none"
+                  rel="noopener"
+                  href="#"
+                  @click="jumpTo($event, property.property)"
+                >
+                  {{ getLabel(property.property) }}
+                </a>
+              </li>
+            </ul>
+            <span v-if="!error.properties">
+              <a
+                class="text-decoration-none"
+                rel="noopener"
+                href="#"
+                @click="jumpTo($event, error.property)"
+              >
+                {{ getLabel(error.property) }}
+              </a>
+            </span>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { toRefs, useAttrs, defineEmits, defineProps, ref } from "vue";
+import $ from "jquery";
+
+const props = defineProps({
+  reports: { type: Object, required: true },
+});
+const $emits = defineEmits(["onJumpTo"]);
+const { reports } = toRefs(props);
+const hideErrors = ref(reports.value.hideErrors)
```

Comment:
There is hideError in "reports" from props. Used toRefs to make the property of hideError "reactive", but the change of value in hideError directly from reports.value will not make the UI reactive. So, added another variable of hideErrors with 'ref'